### DISCUSSION TITLE
How to escape a dot character (`.`) in a template?

### DISCUSSION THREAD

**Discussion by aradalvand** [JanDeDobbeleer/oh-my-posh#2440] [Troubleshoot]
Hi there. I have a segment for .NET that just shows the .NET icon along with the text ".NET" next to it. But in the template, the text .NET gets interpreted as a property interpolation I think, which causes weird results.
```json
{
    "type": "dotnet",
    "style": "diamond",
    "background": "#390f51",
    "foreground": "#dfd0dd",
    "trailing_diamond": "\ue0b4 ",
    "leading_diamond": "\ue0b6",
    "template": " .NET \uE77F " // The "." requires escaping
},
```
This is what it will look like:
![image](https://user-images.githubusercontent.com/26527405/175812426-74738974-8fd6-4d83-8eb1-bd89eabb7149.png)

So, how do I escape the `.` character?

**Comment by JanDeDobbeleer** [comment]
@aradalvand you can't. That notation is reserved for properties. We could allow escaping a literal dot, to ensure we don't replace it (which is what you see).

**Reply by JanDeDobbeleer** [reply]
@aradalvand https://github.com/JanDeDobbeleer/oh-my-posh/pull/2452
